packages/react-core/src/components/Wizard/WizardBody.tsx

```diff
@@ -7,14 +7,26 @@ export interface WizardBodyProps {
   children: any;
   /** Set to true to remove the default body padding */
   hasNoBodyPadding: boolean;
+  /** An aria-label to use for the main element */
+  'aria-label'?: string;
+  /** Sets the aria-labelledby attribute for the main element */
+  'aria-labelledby': string;
+  /** Component used as the primary content container */
+  mainComponent?: React.ReactNode;
 }
 
 export const WizardBody: React.FunctionComponent<WizardBodyProps> = ({
   children,
-  hasNoBodyPadding = false
-}: WizardBodyProps) => (
-  <main className={css(styles.wizardMain)}>
-    <div className={css(styles.wizardMainBody, hasNoBodyPadding && styles.modifiers.noPadding)}>{children}</div>
-  </main>
-);
+  hasNoBodyPadding = false,
+  'aria-label': ariaLabel = null,
+  'aria-labelledby': ariaLabelledBy,
+  mainComponent = 'div'
+}: WizardBodyProps) => {
+  const MainComponent = mainComponent as React.ElementType;
```

Comment:
should the type be ElementType to begin with so you don't have to cast it?

Reply:
Made the change, but isn't won't it be a breaking change? I was asserting to `ElementType` to improve from `any` but to also try and keep from breaking anyone. Lemme know what you think, happy to do whatever we think is best.

Reply:
Oops, nvm, forgot this is a new prop :)